relay-general/src/store/transactions/rules.rs

```diff
@@ -225,14 +203,8 @@ impl TransactionNameRule {
 
     /// Returns `true` if the current rule pattern matches transaction, expected transaction
     /// source, and not expired yet.
-    fn matches(&self, transaction: &str, info: &TransactionInfo) -> bool {
-        let now = Utc::now();
-        info.source
-            .value()
-            .map(|s| s == &self.scope.source)
-            .unwrap_or_default()
-            && self.expiry > now
-            && self.pattern.compiled().is_match(transaction)
+    fn matches(&self, transaction: &str) -> bool {
+        self.expiry > Utc::now() && self.pattern.compiled().is_match(transaction)
```

Comment:
Is there a unit test for to make sure it behaves as we expect for sanitized transactions? 
Maybe I'm thinking about this wrong, but when the new pattern comes in, which will the current url, it will get replaced and already sanitized value will be saved in meta info.

Reply:
Good point, will add some tests.